lms/djangoapps/mobile_api/testutils.py

```diff
@@ -40,6 +61,8 @@ def setUp(self):
         self.password = 'test'
         self.username = self.user.username
 
+        self.prereq_course = CourseFactory.create()
```

Comment:
The MobileAPIMilestonesMixin required ModuleStoreTestCase in order to use CourseFactory.